bin/internal/update_dart_sdk.sh

```diff
@@ -12,7 +12,7 @@
 #
 # -------------------------------------------------------------------------- #
 
-set -e
+set -ex
```

Comment:
did you mean to leave the `-x` here? I think this would break things.

Reply:
I did not.